omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -143,6 +143,8 @@
       "PIP_FIND_LINKS" => "#{build_deps_dir}",
       "CFLAGS" => "-I#{install_dir}/embedded/include -I/opt/mqm/inc",
       "CXXFLAGS" => "-I#{install_dir}/embedded/include -I/opt/mqm/inc",
+      # Some libraries (looking at you, aerospike-client-python) need EXT_CFLAGS instead of CFLAGS.
+      "EXT_CFLAGS" => "-I#{install_dir}/embedded/include -I/opt/mqm/inc",
```

Comment:
If we want `CFLAGS` and `EXT_CFLAGS` to be equal, we may want to update lines 164-166 below, because they update `CFLAGS`:
```
    if linux? || windows?
      nix_build_env["CFLAGS"] += " -std=c99"
    end
```

Reply:
Good point, I'll do that.